build.gradle

```diff
@@ -78,6 +79,9 @@ dependencies {
 
 	// Occlusion Culling (https://github.com/LogisticsCraft/OcclusionCulling)
 	include implementation("com.logisticscraft:occlusionculling:${project.occlusionculling_version}")
+
+	// neu repoparser
+	include implementation("moe.nea:neurepoparser:${project.repoparser_version}")
```

Comment:
Why are we importing and JiJing dependencies that we don't even use?

Reply:
I think the goal is to start using this library? But you're right, we can include it later when we actually use it.

Reply:
you are right i will remove it for now

Reply:
i have commented it out